docker-compose.yml

```diff
@@ -183,6 +183,50 @@ services:
       - shippingservice
     logging: *logging
 
+  # Frotend Tests
+  frontendtests:
+    image: cypress/included:10.3.1-typescript
+    depends_on:
+      - frontend
+    profiles:
+      - tests
+    environment:
+      - CYPRESS_baseUrl=http://${FRONTEND_ADDR}
+      - NODE_ENV=production
+    working_dir: /cypress
+    volumes:
+      - ./src/frontend:/cypress
+
+  # Integration Tests
+  integrationTests:
```

Comment:
We should sync the case for `frontendtests` and `integrationTests`. Maybe `frontendTests`?